Kernel_23/test/Kernel_23/include/CGAL/_test_cls_aff_transformation_2.h

```diff
@@ -577,6 +577,13 @@ _test_cls_aff_transformation_2(const R& )
  assert( ident.homogeneous(1,2) == ident.hm(1,2) );
  assert( gscale.homogeneous(1,1) == gscale.hm(1,1) );
 
+ //equality
+ CGAL::Aff_transformation_2<R> a2(0,1,0,1),
+     a3(0,1,0,1), a4(0,0,1,1);
+ assert(a2 == a3);
+ assert(a3 != a4);
```

Comment:
You should test `Aff_transformation_3` as well.

Reply:
Don't we have to compare quotients for Homogeneous?

Reply:
@afabri the operator== compares the results of homogeneous(i,j), so it is taken into account.